django_tenants/postgresql_backend/introspection.py

```diff
@@ -13,12 +13,15 @@ def __init__(self, cursor, connection):
 
     def __enter__(self):
         self.cursor.execute('SHOW search_path')
-        self.original_search_path = self.cursor.fetchone()[0].split(',')
+        self.original_search_path = [
+            search_path.strip().replace('"', '')
+            for search_path in self.cursor.fetchone()[0].split(',')
+        ]

```

Comment:
Based on the original description I think we need a test that would simulate switching between 2 schemas, at least one of them containing underscore in its name.

Reply:
FTR tests are available in https://github.com/django-tenants/django-tenants/pull/921